lib/model/order.dart

```diff
@@ -0,0 +1,15 @@
+
+part of model;
+
+class Order {
+  Province destination = '' as Province;
+  void resolve(){}
```

Comment:
Making the class abstract will remove the need to have a function body. Subclasses should override resolve.